azure-functions/src/durable/actions.rs

```diff
@@ -0,0 +1,134 @@
+use chrono::{DateTime, FixedOffset};
+use serde::Serialize;
+use serde_json::Value;
+
+#[doc(hidden)]
+#[derive(Debug, Clone, Serialize, PartialEq)]
+#[serde(tag = "actionType", rename_all = "camelCase")]
+pub enum Action {
+    #[serde(rename_all = "camelCase")]
```

Comment:
Does the top-level `rename_all` not perform the same thing?  Just my ignorance with serde and enum serialization.

Reply:
Nope, I was very surprised by this behavior. Looks like a bug